### PR TITLE
March 2022 Agenda

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

**Comment by AngelOnFira** [comment]
A few unknowns:

- Do we have any other potential speakers?
- Should the time slot for the Rust Foundation be longer? Need to check in about length of presentation

File: src/meetings/2022-03-21.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+| Calendar event (with Zoom details)             | [link][cal]              |
+| Social hour calendar event (with Zoom details) | [link][calsh]            |
+| Recording                                      | (link posted after call) |
+
+[see in your time zone]: https://everytimezone.com/s/d733869e
+
+[cal]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NnFqbGt2bHFpOTZjOW9pZWZiczNrc285a2wgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+[calsh]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NzUxcGUwNzFwZ2dzMGw5dW9ub2VraDFqOWogN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+
+## Agenda
```
Comment (nikomatsakis) [inline src/meetings/2022-03-21.md:15]: I believe we also planned to have a "Lang Team Roadmap" update (cc @joshtriplett)

Comment (nikomatsakis) [reply src/meetings/2022-03-21.md:15]: I'm trying to think what a good paragraph would be! Something like

- (15 min) Planning for Rust 2024

Now that Rust 2021 is behind us, it's time to start thinking about Rust 2024! The lang team has been working on a roadmap that proposes various priorities for Rust 2024 and discusses some of the language changes that can help support those priorities. @nikomatsakis and @joshtriplett will discuss the contents of that document. We are interested in broadening this document to cover efforts from other teams as well.

Comment (nikomatsakis) [reply src/meetings/2022-03-21.md:15]: Don't love that but it's the right idea :)

**Review by nikomatsakis — CHANGES_REQUESTED** [review]

File: src/meetings/2022-03-21.md (hunk 1 above)
Comment (AngelOnFira) [reply src/meetings/2022-03-21.md:15]: Paragraph looks good to me, I left it as is :) :+1:

**Pull request merged**